servers/visual_server.h

```diff
@@ -508,6 +508,7 @@ class VisualServer : public Object {
 
 	virtual void particles_set_emitting(RID p_particles, bool p_emitting) = 0;
 	virtual bool particles_get_emitting(RID p_particles) = 0;
+	virtual bool particles_get_inactive(RID p_particles) = 0;
```

Comment:
This is not necessary, emitting may be needed to eventually serialize back the state of the particles.

Inactive is an internal variable that should never be exposed, the reason is that VisualServer, when running on a thread, puts all functions into a command buffer. If data is requested from it in that scenario by calling a function that returns something, it will force all the previous commands to execute and stall rendering, severely affecting performance.

Because of this, internal run-time state should never be requested. For this PR to be merged, just not exposing this function is enough.